lib/Driver/ToolChains/MSVC.cpp

```diff
@@ -1291,26 +1301,38 @@ void MSVCToolChain::AddFortranStdlibLibArgs(const ArgList &Args,
       useOpenMP = true;
   }
 
+  if (!Args.hasArg(options::OPT_nostdlib, options::OPT_nostartfiles)) {
+     if (getDriver().IsFortranMode()) {
+       TC.AddLinkerHelper(Args, CmdArgs, AddMLinker, "/defaultlib:msvcrt");
+     } else if (!getDriver().IsCLMode() && !getDriver().IsFortranMode()) {
+       TC.AddLinkerHelper(Args, CmdArgs, AddMLinker, "/defaultlib:libcmt");
+     }
+  }
+
   CmdArgs.push_back(Args.MakeArgString(std::string("-libpath:") +
                                         D.Dir + "/../lib"));
 
   if (needFortranMain(D, Args)) {
-    CmdArgs.push_back("-defaultlib:flangmain");
-    CmdArgs.push_back("-subsystem:console");
+    TC.AddLinkerHelper(Args, CmdArgs, AddMLinker, "-defaultlib:flangmain");
```

Comment:
This needs to be `/defaultlib:flangmain` for now

Reply:
okay.

Reply:
For the others as well?

Reply:
Yes. for all in `TC.AddLinkerHelper`. This is because `flang2.exe` still doesn't understand the argument after `-linker` is not an option even if it starts with `-`